packages/vc-api/src/v2-vc-api/v2-vc-router.ts

```diff
@@ -8,7 +8,10 @@ export function V2VcRouter(options: any): Router {
   const router = Router();
   router.use(VCApiDocsRouter());
   router.use(VCApiSchemaRouter(options));
-  router.use(apiKeyAuth(options.apiKey));
+  if (options.apiKey) {
+    console.log('Using API key auth');
+    router.use(apiKeyAuth(options.apiKey));
+  }
```

Comment:
Could you double-check this block of code? I noticed that the `apiKeyAuth` function only accepts a single argument, which should be an object containing an `apiKey` field (`{ apiKey: string }`). So I'm not sure if passing the `apiKey` string directly to the function is correct.

<img width="538" alt="image" src="https://github.com/user-attachments/assets/086eb94a-03fd-43a7-a047-fbe60a3c7b42">

Reply:
Thanks @huynguyen-hl, my mistake